src/OpenSage.Game/Client/Drawable.cs

```diff
@@ -484,6 +484,31 @@ public bool GetFirstRenderObjInfo(
         return true;
     }
 
+    private static readonly Dictionary<BodyDamageType, ModelConditionFlag> DamageTypeLookup = new()
```

Comment:
Is it worth using `FrozenDictionary` since this collection is never updated? It takes a bit longer to instantiate, but especially since this is static readonly we'd only have to do it once and the performance is quite a bit better.

Reply:
Good idea, I've done that.